stackrox/stackrox
# ROX-19288: Delete AccessControlNoPermission element

## Description

Follow up task: element is redundant after `resourceAccessRequirements` for `'access-control'` in #7236

## Checklist

## Testing Performed

Even webpack cannot remove conditionally rendered elements when conditions become impossible.

1. `yarn lint` in ui
2. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.css: 0 = 3761772 - 3761772
        total: -905 = 9975268 - 9976173
    * `ls -al build/static/js/*.js | wc -l`
        0 = 80 - 80 files
3. `yarn start` in ui

### Manual testing

1. Visit /main/access-control with NO_ACCESS for Access resource.
    * See that `Body` renders generic `NotFoundPage` element.
    * See absence of data requests
    * See absence of request for Access Control bundle in Network pane.
        ![access-control](https://github.com/stackrox/stackrox/assets/11862657/2834f4c0-b5d4-43e9-abd3-623797ef332f)
